tests/test_sra.py

```diff
@@ -17,6 +18,9 @@ def test_empty_sra_file_raises_exception():
     assert not task.output().exists()
     assert not task.complete()
 
+def test_sra_file_with_missing_header():
+    assert luigi.build([DownloadSraExperiment(srx='SRX12752257')], local_scheduler=True)
+
```

Comment:
This test does not work inside Tox because it somehow overrides the home directory where the NCBI toolkit retrieves its download path.

Reply:
It's actually working on the CI, but that's only due to the user allowed to create a `/ncbi` directory...